client/src/tailwind.css

```diff
@@ -26,6 +26,14 @@
   height: 761px;
 }
 
+.contacts-block {
+  width: 760px;
```

Comment:
I wonder how it looks on screens more narrow than 760px + 760px + 3rem

Reply:
fixed